src/components/Tooltip/Tooltip.js

```diff
@@ -215,13 +233,16 @@ Tooltip.propTypes = {
   /** String for overriding default tooltip box styles. You can, for example override the white background color with this. */
   popperBoxStyles: PropTypes.string,
   /** Tooltip description */
-  details: PropTypes.string.isRequired,
+  details: PropTypes.oneOfType([PropTypes.string, PropTypes.element])
+    .isRequired,
   /** String that sets what Element the tooltip events should trigger against, can be `'viewport'`, `'scrollParent'` or`'window'`*/
   boundariesElement: PropTypes.oneOf(Object.values(Tooltip.BOUNDARIES_ELEMENT)),
   /** The Modal's children */
   children: PropTypes.node,
   /** Classes to apply to root element */
   className: PropTypes.string,
+  /** Indicates whether to use soft edges*/
+  useSoftEdges: PropTypes.bool,
```

Comment:
In my mind, the `use` prefix has been claimed by React Hooks, and this opens up a channel for confusion.